db/db.go

```diff
@@ -60,7 +61,11 @@ func (d *Database) View(fn func(txn *badger.Txn) error) error {
 	if d.duringTransaction() {
 		return fn(d.txn)
 	}
-	return d.store.Badger().View(fn)
+	err := d.store.Badger().View(fn)
+	if err != nil {
+		return errors.WithStack(err)
```

Comment:
Add a stack trace here, the very first part of our codebase which this error touches. Shockingly, this seems to be the best we can do in golang, if you want full stack traces you have to only use libraries which attack stack traces to their errors.